compiler/p/codegen/PPCOpsDefines.hpp

```diff
@@ -926,7 +926,17 @@ FORMAT_VRS_D34_RA_R,
 // |     | XS        | RA       | d1                              |
 // | 0   | 5         | 11       | 16                              |
 // +-----+-----------+----------+---------------------------------+
-FORMAT_XS5_D34_RA_R
+FORMAT_XS5_D34_RA_R,
+
+
+// Formats for instructions with a VRT field encoding the target VSX register and field
+// encoding an 8-bit immediate value:
+//
+// +------+----------+------+----------------+--------------------+
+// |      | T        | 0    | IMM8           |                    |
+// | 0    | 6        | 11   | 13             | 21                 |
+// +------+----------+------+----------------+--------------------+
```

Comment:
This diagram and description don't quite seem to quite match how this format is actually encoded. The XT field denoting the target register is actually split between two parts of the instruction:

```suggestion
// Formats for instructions with an XT field encoding the target VSX register and an IMM8 field
// encoding an 8-bit immediate value:
//
// +------+----------+------+----------------+---------------+----+
// |      | XT       |      | IMM8           |               | XT |
// | 0    | 6        | 11   | 13             | 21            | 31 |
// +------+----------+------+----------------+---------------+----+
```